File: Algorithms/p119_Pascal's_Triangle_II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> res(rowIndex+1,1);
        int old_v;
        int new_v;
        for(int i=1;i<=rowIndex;++i){
            old_v = res[0];
            for(int j=1;j<i;++j){
                new_v = old_v+res[j];
                old_v = res[j];
                res[j] = new_v;
            }
        }
        return res;
    }
};

File: Algorithms/p119_Pascal's_Triangle_II/p119_Pascal's_Triangle_II.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        res = [1]*(rowIndex+1)
        for i in range(1,rowIndex+1):
            old_v = res[0]
            for j in range(1,i):
                res[j],old_v = old_v+res[j],res[j]
        return res
